generic_pva.xml.fdm_material

```diff
@@ -128,6 +131,9 @@ Generic PVA profile. The data in this file may not be correct for your specific
             <hotend id="CC 0.6">
                 <setting key="hardware compatible">no</setting>
             </hotend>
+            <hotend id="DD 0.4">
```

Comment:
I know this is not the correct location, but the version number `<version>25</version>` should be increased for each of these files.